### DISCUSSION TITLE
Upgrade to v5.0 killed my sidebery setup

### DISCUSSION THREAD

**Discussion by up-bpersons** [mbnuqw/sidebery#1245] [General]
CSS styling no longer works as before, despite no change to the styles. Anyone else have this issue?

**Comment by zelch** [comment]
Are you coming from v4, or from one of the v5 release candidates?

**Comment by Elementari** [comment]
I also experienced some configuration data loss, and it caused considerable interruption of my workflow.
However the joy over the new release was much greater.

I would like to suggest that major upgrades could be rolled out as follows:
A pure "notication update" is rolled out 2 weeks prior to the actual update. The "notification update" will inform the user that a major automatic update is about to be rolled out on (date) at (time) (timezone). 
The user can then disable automatic updates for Sideberry in FF Addons config (perhaps provide a quick explanation to users how to go there), thereby having full control about the update process.

I was anticipating that the v5 update would come with some config data loss so I had taken some screenshots of my configuration as a quick reference a week ago, thankfully (coincidence).

I did not experience tab data loss.

**Reply by mbnuqw** [reply]
> A pure "notication update" is rolled out 2 weeks prior to the actual update. The "notification update" will inform the user that a major automatic update is about to be rolled out on (date) at (time) (timezone).
The user can then disable automatic updates for Sideberry in FF Addons config (perhaps provide a quick explanation to users how to go there), thereby having full control about the update process.

Yeah, I thought about it too late... Sorry.

**Comment by mbnuqw** [comment]
https://github.com/mbnuqw/sidebery/issues/1241#issuecomment-1732348639
